### PR TITLE
Suspicious behaviours related to  SOURGUM

### PR DESCRIPTION
**Pull request by hieuttmmo**
Suspicious behaviours related to an actor tracked by Microsoft as SOURGUM
MSTIC Blog: https://www.microsoft.com/security/blog/2021/07/15/protecting-customers-from-a-private-sector-offensive-actor-using-0-day-exploits-and-devilstongue-malware/

### CODE REVIEW

**Comment by hieuttmmo** [comment]
Although the rule 'Monitoring For Persistence Via BITS' by Sreeman already cover the HTTP Download leveraging the Bitsadmin.exe but the "|re" modifier will not work well in many backends.
This rule will make it much easier for backends to handle with basic logic conditions

File: rules/windows/process_creation/sysmon_apt_sourgrum.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    selection3:
+        Image|contains:
+            - 'windows\system32\filepath2'
+            - 'windows\system32\ime'
+    registry_command:
+        CommandLine|contains:
+            - 'reg add'
+    registry_key:
+        CommandLine|contains: 
+            - 'HKEY_LOCAL_MACHINE\software\classes\clsid\{7c857801-7381-11cf-884d-00aa004b2e24}\inprocserver32'
```
Comment (frack113) [inline rules/windows/process_creation/sysmon_apt_sourgrum.yml:39]: Add: 
` - HKLM\SOFTWARE\Classes\CLSID\{7C857801-7381-11CF-884D-00AA004B2E24}\InProcServer32' 
As in  microsoft ref

File: rules/windows/process_creation/sysmon_apt_sourgrum.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            - 'HKEY_LOCAL_MACHINE\software\classes\clsid\{cf4cc405-e2c5-4ddd-b3ce-5e7582d8c9fa}\inprocserver32'
```
Comment (frack113) [inline rules/windows/process_creation/sysmon_apt_sourgrum.yml:40]: Add:
`- HKLM\SOFTWARE\Classes\CLSID\{CF4CC405-E2C5-4DDD-B3CE-5E7582D8C9FA}\InprocServer32'
As in microsoft ref

File: rules/windows/process_creation/sysmon_apt_sourgrum.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+title: SOURGUM Actor Behaviours
+id: 7ba08e95-1e0b-40cd-9db5-b980555e42fd
+description: Suspicious behaviours related to an actor tracked by Microsoft as SOURGUM
+author: 
+    - MSTIC
+    - FPT.EagleEye
+status: experimental
+level: high
+references:
+    - sha256=c299063e3eae8ddc15839767e83b9808fd43418dc5a1af7e4f44b97ba53fbd3d
```
Comment (frack113) [inline rules/windows/process_creation/sysmon_apt_sourgrum.yml:10]: It is not human readable reference, this IOC is in the microsoft ref, and it is not use in the rule.
I don't think this is helpful in understanding the rule.

**Review by frack113 — COMMENTED** [review]
Hi,
I've check with the documention.
I've not test with a POC and a real event log.

File: rules/windows/process_creation/sysmon_apt_sourgrum.yml (hunk 1 above)
Comment (hieuttmmo) [reply rules/windows/process_creation/sysmon_apt_sourgrum.yml:39]: Cool bro. Thanks for that

**Comment by Neo23x0** [comment]
The references should always be links - could you change the first reference to e.g. 
https://www.virustotal.com/gui/file/c299063e3eae8ddc15839767e83b9808fd43418dc5a1af7e4f44b97ba53fbd3d/detection

**Comment by hieuttmmo** [comment]
Sure, just updated it. Thanks for your commend Florian

**Pull request merged**